doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -66,6 +78,8 @@ and pretty-printer of Coq already know how to deal with the syntactic
 expression (see :ref:`ReservingNotations`), explicit precedences and
 associativity rules have to be given.
 
+.. todo PR above is correct?
```

Comment:
I don't know but it seems to be generally the case.

Reply:
It's sort-of tautologically true.  The question is "when does Coq know how to deal with the syntactic expression?"  The obvious case is "you gave `Reserved Notation` for it", but there are a few others, such as when there is just a symbol (`Notation "'new-symbol'" := O.` works and emits "Setting notation at level 0.") or a couple symbols (`Notation "'new-symbol' 'another-symbol'" := O.`).  I think those are the only special cases.  (Okay, I guess technically if you declare a notation `only parsing` and you have a `Reserved Notation` that is also `only parsing` then the pretty-printer doesn't know how to deal with the expression, but it works anyway because it doesn't need to.)